### PR TITLE
RDM Responder improvements

### PR DESCRIPTION
**Pull request by nomis52**
- Fix a mute / unmute bug
- Fix all subdevice handling
- Indicate identify / mute status with LEDs
- Dump counters from the console

### CODE REVIEW

File: firmware/src/message_handler.c [hunk 1]
```diff
@@ -197,6 +197,7 @@ void MessageHandler_HandleMessage(const Message *message) {
         SendMessage(message->token, message->command, RC_BUFFER_FULL, NULL, 0);
       }
       break;
+      break;
```
Comment (peternewman) [inline firmware/src/message_handler.c]: Remove the duplicate break.

File: firmware/src/rdm_responder.c [hunk 2]
```diff
@@ -54,9 +57,20 @@ typedef struct {
   RDMResponderSendCallback send_callback;
   uint8_t uid[UID_LENGTH];
   uint8_t queued_message_count;
-  bool is_muted;
   uint16_t dmx_start_address;
 
+  // Mute Settings
+  bool is_muted;
+  CoarseTimer_Value mute_timer;
+  PORTS_CHANNEL mute_port;  //!< The port to use for the identify signal.
```
Comment (peternewman) [inline firmware/src/rdm_responder.c]: Fix the comment.

File: firmware/src/rdm_responder.c [hunk 3]
```diff
@@ -54,9 +57,20 @@ typedef struct {
...
+  PORTS_BIT_POS mute_bit;  //!< The port bit to use for the identify signal.
```
Comment (peternewman) [inline firmware/src/rdm_responder.c]: Again.

File: firmware/src/system_config/default/system_settings.h [hunk 4]
```diff
@@ -17,6 +17,21 @@ extern "C" {
 #endif
 
 /**
+ * @brief The port used used by the RDM Responder.
```
Comment (peternewman) [inline firmware/src/system_config/default/system_settings.h]: SPAG used

**Comment by peternewman** [comment]
LGTM, a few comments to address.

File: firmware/src/message_handler.c (hunk 1 above)
Comment (nomis52) [inline firmware/src/message_handler.c]: Done

File: firmware/src/rdm_responder.c (hunk 2 above)
Comment (nomis52) [inline firmware/src/rdm_responder.c]: Done

File: firmware/src/rdm_responder.c (hunk 3 above)
Comment (nomis52) [inline firmware/src/rdm_responder.c]: Done

File: firmware/src/system_config/default/system_settings.h (hunk 4 above)
Comment (nomis52) [inline firmware/src/system_config/default/system_settings.h]: Fixed

**Pull request merged**